cases_high_level_simulation/test_3d_unit_position_based_bc/position_based_boundary_conditions.cpp

```diff
@@ -198,16 +198,16 @@ TEST(StructuralSimulation, TranslateSolidBodyTuple)
 		physical_viscosity,
 		{}
 	};
-	Vecd translation_vector = Vec3d(0.0, 0.0, 0.1); //changed to be more like PositionSolidBodyTuple
+	Vecd translation_vector = Vec3d(0.0, 0.0, 0.1);
 	input.translation_solid_body_tuple_ = { TranslateSolidBodyTuple(0, end_time * 0.32, end_time, translation_vector) };
 
 	//=================================================================================================//
 	TestStructuralSimulation sim (input);
 	sim.TestRunSimulation(end_time);
 	//=================================================================================================//
 
-	StdLargeVec<Vecd>& pos_0 = sim.Get_translation_solid_body_()[0]->GetParticlePos0();
-	StdLargeVec<Vecd>& pos_n = sim.Get_translation_solid_body_()[0]->GetParticlePosN();
+StdLargeVec<Vecd>& pos_0 = sim.Get_solid_body_list_()[0].get()->getElasticSolidParticles()->pos_0_;
```

Comment:
what is the first get?

Reply:
to get the pointer to access the member
I can't access it directly